custom/icds/tests/test_ls_aggregate_performance.py

```diff
@@ -37,6 +37,7 @@ def test_report_parsing(self, days_open, weighed, thr, visits):
         thr.return_value = ET.fromstring(self.get_xml('thr_fixture'))
         visits.return_value = ET.fromstring(self.get_xml('visit_fixture'))
         indicator = LSAggregatePerformanceIndicator('domain', 'user')
+        self.addCleanup(indicator.clear_caches)
```

Comment:
(removed this, turns out it made the test fail since it wasn't actually creating any users anyway)